src/controllers/login.js

```diff
@@ -0,0 +1,31 @@
+const bcrypt = require('bcryptjs');
+const getData = require('../../src/database/queries/getData');
+const pageTitle = require('../views/helpers/makeTitle');
+
+exports.get = (request, response) => {
+  response.render('login', {
+    js: ['/helpers/collectData', 'login'],
+    css: ['login'],
+    layout: 'login_signup',
+    title: pageTitle.titleName('login'),
+  });
+};
+exports.post = (request, response, next) => {
+  getData.checkEmail(request.body.email.trim())
+    .then((res) => {
```

Comment:
what about anyone trying to log in from the browser itself  
I mean where the validation !!